aten/src/ATen/gen.py

```diff
@@ -43,6 +43,18 @@
     '--rocm',
     action='store_true',
     help='reinterpret CUDA as ROCm/HIP and adjust filepaths accordingly')
+parser.add_argument(
+    '--op_registration_whitelist',
+    nargs='*',
+    help='filter op registrations by the whitelist (if set)')
```

Comment:
It would be very helpful to give an example valid entry for the whitelist. Do we whitelist full names with overloads `add.Tensor`? Or just `add`?